examples/README.md

```diff
@@ -0,0 +1,28 @@
+# HWP CLI Examples
+
+This directory contains examples demonstrating how to use the HWP CLI package.
+
+## Available Examples
+
+### Basic CLI Usage
```

Comment:
As we add examples I think having these long explanations for each will be unhelpful. Maybe a simple list with a brief description? e.g.

- `wp-env` cli: This example shows how to use wp-env to manage a local WP instance from the CLI.